keras_cv/models/object_detection/retina_net/retina_net_inference_test.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2022 The KerasCV Authors
```

Comment:
Any reason to have this as a standalone test file?

Reply:
Basically just QOL for me, there are a lot of RetinaNet tests.  an additional aspect is timeouts